### PR TITLE
Add logpath of static vhost

### PR DESCRIPTION
**Pull request by MattiasGees**

### CODE REVIEW

File: templates/etc/nginx/sites-available/static.conf.erb [hunk 1]
```diff
@@ -24,7 +24,8 @@ server {
   server_name <%= @name %> <% @server_aliases.each do |server_alias| -%><%= server_alias %> <% end -%>;
   client_max_body_size <%= @client_max_body_size %>;
 
-  access_log  /var/log/nginx/<%= @name %>.access.log;
+  access_log <%= @log_path %>/access.log;
```
Comment (iuriaranda) [inline templates/etc/nginx/sites-available/static.conf.erb]: What if `@log_path` is undef? Shouldn't you fallback to the other path?

Comment (MattiasGees) [inline templates/etc/nginx/sites-available/static.conf.erb]: I rather fail then :). We need to implement this on the vhosts as well then.

Comment (MattiasGees) [inline templates/etc/nginx/sites-available/static.conf.erb]: Changed it to add the default log path of nginx

**Comment by iuriaranda** [comment]
:+1:

**Comment by venturel** [comment]
👍

**Pull request merged**